### PR TITLE
[Tour Of Beam] return taskSnippetId/solutionSnippedId

### PR DESCRIPTION
**Pull request by eantyshev**
We have a convention between Playground and Tour Of Beam that ToB examples and snippets (and files and pc_objects too)
have the following key:
`<origin>_<SDK>_<name>` where
```
origin = "TB_EXAMPLES"
SDK = "SDK_JAVA" | "SDK_PYTHON" | "SDK_GO" | "SDK_SCIO"
name = /beam-playground/name
```
name is from an embedded yaml tag in a code example
duplicated also as taskName or solutionName in `learning-content/.../unit-info.yaml`

addresses #23421

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: learning/tour-of-beam/backend/internal/entity.go [hunk 1]
```diff
@@ -33,8 +33,6 @@ type Unit struct {
 	Hints             []string `json:"hints,omitempty"`
 	TaskSnippetId     string   `json:"taskSnippetId,omitempty"`
 	SolutionSnippetId string   `json:"solutionSnippetId,omitempty"`
-	TaskName          string   `json:"-"`
-	SolutionName      string   `json:"-"`
```
Comment (eantyshev) [inline learning/tour-of-beam/backend/internal/entity.go:37]: clean up the model from intermediate fields

File: learning/tour-of-beam/backend/internal/fs_content/load.go [hunk 2]
```diff
@@ -38,50 +38,34 @@ const (
...
 	Content []string `yaml:"content"`
 }
 
 type learningModuleInfo struct {
-	Id         string
-	Name       string
-	Complexity string
+	Id         string   `yaml:"id"`
+	Name       string   `yaml:"name"`
+	Complexity string   `yaml:"complexity"`
```
Comment (eantyshev) [inline learning/tour-of-beam/backend/internal/fs_content/load.go:48]: It worked, but now we set yaml tags to make this code explicit

**Comment by olehborysevych** [comment]
lgtm

**Comment by olehborysevych** [comment]
R: @damccorm

**Review by damccorm — APPROVED** [review]
LGTM, thanks!

**Pull request merged**